docs/example-relationship-definition.yml

```diff
@@ -0,0 +1,60 @@
+# a list of relationships defined in a file under the entity in which the telemetry is sourced
+relationships:
+  # implicit convention is to provide a nrql query that you're trying to replicate here. This helps in
+  # documentation and also during PR approval process.
+   
+  # select entityGuid, peer.hostname from Span where entityGuid is not null and peer.hostname is not
+  # null
+   
+  # what provider is the telemetry coming from. This currently is not used for much other than
+  # providing browsing information 
+  - instrumentation.provider: newrelic
+    # what NRDB event type should we filter telemetry from. eg Transaction, AjaxRequest, Span or a
+    # custom event type
+    eventType: Span
+    # there is a predefined set of relationship types. this is the connecting edge between the
+    # source and target nodes. eg CALLS, HOSTS, CONTAINS, IS
+    relationship: CALLS
+    # information of how to formulate the target node of the relationship 
+    target:
```

Comment:
Source and target are mutually exclusive, so I'd show this as 

```
[source | target]:
  ...
```

Adapting the explanations in the comments.

Reply:
done!